packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -125,7 +158,7 @@ async function handleDeleteContext(contextName: string) {
         <div class="grow flex-column divide-gray-900 text-[var(--pd-invert-content-card-text)]">
           <div class="flex flex-row">
             <div class="flex-none w-36">
-              {#if $kubernetesContextsState.get(context.name)?.reachable}
+              {#if isContextReachable(context.name, experimentalStates)}
```

Comment:
I would say, that from UI side (html part), we should not call or do any computation at the display time, so no call to a map or a function, the data providing already all the conditions for the rendering.

so IMHO it should be part of the data we're receiving, so only

```svelte
{#if someObjectToDisplay.reachable}
   ....
{/if}
```svelte

and not utility function called at various places in the .svelte/html part

Reply:
I moved the logic out of the html